tap_trello/schemas/cards.json

```diff
@@ -10,14 +10,95 @@
         "string"
       ]
     },
-    "checkItemStates": {
+    "customFieldItems": {
       "type": [
         "null",
         "array"
       ],
       "items": {
+        "type": [
+          "null",
+          "object"
+        ],
+        "properties": {
+          "id": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idValue": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idCustomField": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "idModel": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "modelType": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "value": {
+            "type": [
+              "null",
+              "object"
+            ],
+            "properties": {
+              "checked": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "date": {
+                "type": [
+                  "null",
+                  "string"
+                ]
+              },
+              "number": {
+                "type": [
+                  "null",
+                  "string"
```

Comment:
This should be a number type, I think. We really need to get `singer.decimal` supported across python taps...